### ISSUE TITLE
CSS not applying in master branch

### ISSUE THREAD

**Issue by msavastano** [liberodark/ODrive#129]
**Describe the bug**
Css and Font awesome icons do not apply in pre-release branch

**To Reproduce**
Open Preferences screen
![master](https://user-images.githubusercontent.com/2404371/63305730-07127f80-c2b6-11e9-911b-323e49dca5d5.png)

**Expected behavior**
Expect same as latest release
![0 2 2](https://user-images.githubusercontent.com/2404371/63305747-1691c880-c2b6-11e9-845c-eae5dcd05a84.png)

**Desktop (please complete the following information):**
 - OS: Linux Mint
 - Version 19.2 Cinnamon

**Additional context**
Looks like webpack plugin change caused regression issue

**Comment by msavastano** [comment]
#131

**Comment by liberodark** [comment]
Yes have see that thanks for your pull request im go to see that !
